src/api/README.md

```diff
@@ -55,15 +55,24 @@ This function takes user-provided data and uses it to update an exiting item in
 | Parameter  | Type     | Description                                                     |
 | ---------- | -------- | --------------------------------------------------------------- |
 | `listPath` | `string` | The Firestore path of the list to which the item will be added. |
-| `item`     | `string` | The item object.                                                |
+| `item`     | `object` | The item object.                                                |
```

Comment:
I love that we're making updates here! I might suggest that we change `item` from just `object` to include the fields that we are anticipating on that object. You can do that like this:

```
| `item.name`     | `string` | The name of your item                                             |
(ect etc)
```